requirements.txt

```diff
@@ -5,7 +5,7 @@ async-timeout==3.0.1
 beautifulsoup4==4.9.3
 boto3==1.16.36
 botocore==1.19.36
-certifi==2020.12.5
+certifi==2020.6.20
```

Comment:
Wherever possible, you should document in this file why each package is pinned to a particular version.  If a package does not need to be pinned to a specific version, let it fly free!

Reply:
I have added comments to requirements that cannot changed due to gophish version 0.5.1

Reply:
@zenine07 Are you able to remove the specific versions pinned to all of those other requirements (the ones not required by gophish)?  Or change them so that they apply to versions greater than or equal to the versions specified?

Reply:
@dav3r I'd be willing to do ~= (minor updates) for all other requirements. I don't really trust allowing major updates.

Reply:
I'll take it- that is an improvement over pinning to an exact version.  Our mode of operation has typically been to just roll with new versions and only pin when we find a problem.

Reply:
I'll at least try >= for now, I believe the tests we have wrote will find any issues if there are any, I've just never liked trusting backwards-compatibility in major updates.

Reply:
@dav3r I have added >= to all of the requirements. All of the automated tests were passing and also ran a few manual tests of my own and everything looks good using >=. I was able to get away with pinning specific versions as well.